src/components/ReservationCard/ReservationCard.jsx

```diff
@@ -0,0 +1,58 @@
+import styles from "./ReservationCard.module.css";
+import CostsSummary from "../CostsSummary/CostsSummary";
+
+function ReservationCard() {
+  const checkInDate = new Date("2025-01-01");
+  const checkOutDate = new Date("2025-01-16");
+  const pricePerNight = 146;
+  const cleaningFee = 10;
+  const airbnbServiceFee = 10;
+
```

Comment:
To achieve full functionality, we also need to add
const longStayDiscount = 30;
const nightsCountForDiscount = 5;
so that the discount is displayed in the CostsSummary component.

Reply:
Done!

Reply:
Thanks, I did!